Allow uploading images from a url

rzane opened the pull request:
Carrierwave allows you to upload images from a url by simply surrounding the image parameter name with "remote" and "url". This PR just adds that input as well as a button to submit.

I don't speak Portugese, so I hope Google Translate was correct!

volmer commented on app/controllers/bootsy/images_controller.rb:
```diff
@@ -41,8 +41,8 @@ def destroy
-    def find_gallery
-      ImageGallery.find(params[:image_gallery_id])
+    def set_gallery
+      @gallery = ImageGallery.find(params[:image_gallery_id])
```
:heart:

volmer commented on app/views/bootsy/images/_new.html.erb:
```diff
@@ -4,5 +4,14 @@
+    </span>
+  </div>
+
+  <span>&nbsp;&nbsp;or&nbsp;&nbsp;</span>
```
This padding should be set using css. We also need to i18n the text.

volmer commented on features/upload_image.feature:
```diff
@@ -11,6 +11,11 @@ Feature: Upload an image
     Then I see the thumbnail "test.jpg" on the image gallery
 
+  Scenario: Upload a valid image from a URL
+    When I fill in "image[remote_image_file_url]" with "http://placehold.it/350x150"
```
Is this test performing an actual request?

volmer commented:
Hey @rzane, thanks so much for the PR. It looks really good, I've just added a few comments. I also think it would be nice to have a validation somehow, in case someone enters an incorrect URL or something.

rzane commented:
You're very welcome. Thanks for an easy-to-integrate wysiwyg editor!

The test was performing an actual http request. Forgot about that! I added the `sham_rack` gem to stub the HTTP interaction. I also added translations for 'or' and fixed the padding. I added validation in my other PR (#111).

volmer commented:
Many, many thanks for this!
